packages/11ty/_includes/components/object-filters/object-card/object-image.webc

```diff
@@ -9,23 +9,37 @@
 const alt = ({ data }) => typeof data.thumbnail === 'object' ? data.thumbnail.alt : '';
 
 /**
- * Get object image `src` from `figures` or `thumbnail` properties
+ * Returns the served asset location for this image
+ * accounting for fully qualified asset URLs
  */
-const src = ({ data }) => {
-  const { figures } = data;
-  const thumbnail = figures
-    ? figures.find(({ mediaType }) => mediaType === 'image').thumbnail
-    : data.thumbnail
+const assetSrc = (srcPath) => {
+  const regexp = new RegExp(/^(https?:\/\/|\/iiif\/)/)
+  const { imageDir } = this.config.figures
+  return regexp.test(srcPath)) ? srcPath : `${imageDir}/${srcPath}`
+}
 
-  const src = typeof thumbnail === 'object'
-    ? thumbnail.src
-    : thumbnail
+/**
+ * Get object image `src` from `figures` or `thumbnail` properties.
+ */
+const src = ({ data }) => {
+  const { figures, thumbnail } = data
+  const figure = figures?.find(({ mediaType }) => mediaType === 'image')
+  
+  let value = ''
+  switch (true) {
+   case thumbnail: 
+      value = figure; 
+      break;
+   case figure && figure.thumbnail: 
+      value = figure.thumbnail; 
+      break;
+   case figure: 
+      value = figure.src; 
+      break;
```

Comment:
❓ this might be either
```
   case figure && figure.thumbnail: 
      value = figure.thumbnail; 
      break;
   case figure && figure.src: 
      value = figure.src; 
      break;
```
or more succinctly
```suggestion
   case figure: 
      value = figure.thumbnail || figure.src; 
      break;
```